tools/config/go-templates/atlas_client.mustache

```diff
@@ -1,4 +1,4 @@
-{{! X-XGEN-CUSTOM }}package {{packageName}} // import "go.mongodb.org/atlas/mongodbatlas/v2"
```

Comment:
would this comeback with the new url?

Reply:
I do not understand this syntax (it was inspired by v1 api) thus removed it. 
Can restore that now.

Reply:
when using vanity urls both `import "go.mongodb.org/atlas-sdk/admin"` and `import "https://github.com/mongodb/atlas-sdk-gok/admin" are valid, this comment `// import "go.mongodb.org/atlas/mongodbatlas/v2"` throws a nice error when trying to use anything but the vanity url

Reply:
Nice catch. Restoring that now.